docs/contributing-prerequisites.md

```diff
@@ -0,0 +1,75 @@
+# Before submitting your Operator
+
+> **Important:** "First off, thanks for taking the time to contribute your Operator!"
+
+## A primer to Kubernetes Community Operators
```

Comment:
This section is only relevant to k8s operators. Please remove it or replace it with ocp operators.